.github/workflows/docs.yml

```diff
@@ -16,16 +16,13 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+    - name: Install
+      shell: bash -l {0}
+      run: source continuous_integration/scripts/install.sh
+    - name: Install documentation-building requirements
```

Comment:
Can you add a blank line before each new group.  Readability.  It's a preference of mine.

Reply:
Also, just learned the hard way to add a `shell` specifier before `run`, to control which shell for the commands to follow.  With no spec, it's `sh -e {0}`.  If conda is used (as hklpy needs it), then `shell: bash -l {0}` should be specified.

My advice on shells comes from the README of this action: https://github.com/conda-incubator/setup-miniconda/tree/8246c3d5d22bc5ca3416ddf4f74b77c2fb1d33d0#IMPORTANT